packages/e2e/app/src/pages/GiftCardsSessions/GiftCardsSessions.js

```diff
@@ -19,7 +19,11 @@ const initCheckout = async () => {
         clientKey: process.env.__CLIENT_KEY__,
         showPayButton: true,
         session,
-        _translationEnvironment: 'local',
+        _environmentUrls: {
```

Comment:
Just double checking I understand correctly. This is the new 'remote', configuration?
If so indeed I think this is a better approach then before.